lib/lib.es2016.intl.d.ts

```diff
@@ -0,0 +1,40 @@
+/*! *****************************************************************************
+Copyright (c) Microsoft Corporation. All rights reserved.
+Licensed under the Apache License, Version 2.0 (the "License"); you may not use
+this file except in compliance with the License. You may obtain a copy of the
+License at http://www.apache.org/licenses/LICENSE-2.0
+
+THIS CODE IS PROVIDED ON AN *AS IS* BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, EITHER EXPRESS OR IMPLIED, INCLUDING WITHOUT LIMITATION ANY IMPLIED
+WARRANTIES OR CONDITIONS OF TITLE, FITNESS FOR A PARTICULAR PURPOSE,
+MERCHANTABLITY OR NON-INFRINGEMENT.
+
+See the Apache Version 2.0 License for specific language governing permissions
+and limitations under the License.
+***************************************************************************** */
+
+
+
+/// <reference no-default-lib="true"/>
+
+
+declare namespace Intl {
+
+    /**
+     * [Unicode BCP 47 Locale Identifiers](https://unicode.org/reports/tr35/#Unicode_Language_and_Locale_Identifiers) definition.
+     *
+     * [MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl#locales_argument).
+     *
+     * [Wikipedia](https://en.wikipedia.org/wiki/IETF_language_tag).
+     */
+    type UnicodeBCP47LocaleIdentifier = string;
```

Comment:
This would clash and cause compiler errors with [same type](https://github.com/microsoft/TypeScript/pull/39661/files#diff-14c3d4ab96c3bacfa5648a1cc95673ebR9) in https://github.com/microsoft/TypeScript/pull/39661 - which might mean that the one in 2020 should get removed

Reply:
yup it should actually be in as far as es5. I'm planning to make another PR that fix that up as well